server/test/controllers/admin/AdminProgramControllerTest.java

```diff
@@ -266,10 +266,7 @@ public void update_overwritesExistingProgram() throws Exception {
     Result redirectResult = controller.index(addCSRFToken(Helpers.fakeRequest()).build());
     assertThat(contentAsString(redirectResult))
         .contains(
-            "Create new program",
```

Comment:
Note to the reviewer: This one wasn't obvious. But the reason the "Existing one" (e.g. internal admin program name) existed prior to this change was because it was shown in the modal. Now that we use the external-facing name everywhere in the program list view, it's no longer included.